rexmex/__init__.py

```diff
@@ -1,4 +1,2 @@
-from rexmex import dataset, metricset, scorecard, utils
-from rexmex.metrics import classification, coverage, ranking, rating
-
-all = ["scorecard", "dataset", "rexmex.metrics.*", "metricset", "utils"]
```

Comment:
it might be the case that you wanted `__all__`, but the `"rexmex.metrics.*"` entry would not be valid there